### ISSUE TITLE
disable-comments: Block comments (/* */) not detected

### ISSUE THREAD

**Issue by chrismlittle123** [chrismlittle123/check-my-toolkit#138]
## Description

The `findCommentStart` function only detects `//` style line comments for JavaScript/TypeScript. It does not handle `/* */` block comments at all.

## File

`src/code/tools/disable-comments.ts` (lines 143-168 in dist)

## Reproduction

```typescript
/* eslint-disable no-unused-vars */
const x = 1;
/* eslint-enable */
```

## Expected Behavior

The `eslint-disable` pattern should be detected inside the block comment.

## Actual Behavior

Block comment disable directives are not detected because only `//` markers are recognized.

## Impact

Developers using block-style disable comments will have them go undetected, undermining the purpose of the disable-comments checker.

## Suggested Fix

Extend `findCommentStart` to also detect `/*` block comment markers and handle multi-line block comments appropriately.
